code/cellular_automaton/src/von_neumann_cellular_automata/README.md

```diff
@@ -1,7 +1,37 @@
 # cosmos
 > Your personal library of every algorithm and data structure code that you will ever encounter
 
-# Cellular Automaton
+# Von Neumann Cellular Automaton
+
+Jonh Von Neumann worked for years on a simple model for a logical cellular automaton with the purpose to build a self-replication machine. Wich logical organization an automaton will need to be able to control itself in a way that it reproduces itself? To answer that, Von Neumann thought on logical operations and states with different objectives, capable to construct and deconstruct other states for their own. This lead to Von Neumann Cellular System.
+
+## States
+
+To maintain your simplicity, Von Neumann limited your automaton to only 29 states, is that the minimum to build a self-replication machine. This states can be divided into four groups:
+
+ - The **blank state** or unexcitable state (**U**) is the basic and initial state, when it is excited it will become one of the transmission states or a confluent state. If another state has been deconstructed, that cell returns to the blank state.
+
+ - The **transmission states** behave like a conductor wire, where pulses and signals can move from a place to the other. They are sub-divided in ordinary transmission (represented by simple arrows in the four directions: → ↓ ← ↑) and special transmission (represented by double arrows). This separation is justified because of the deconstruct mechanism which will be discussed later. Each one of the directions has two states, the unexcited and excited one. Totalizing in all, we have 16 transmission states which transport binary pulses to the cell that it is pointing for. They can receive this signals from other transmission state or from a confluent state.
```

Comment:
```suggestion
 - The **transmission state** behaves like a conductor wire, where pulses and signals can move from one place another. They are sub-divided into ordinary transmission (represented by simple arrows in the four directions: → ↓ ← ↑) and special transmission (represented by double arrows). This separation is justified because of the deconstruction mechanism which will be discussed later. Each one of the directions has two states, the unexcited and excited one. All in all, we have 16 transmission states which transport binary pulses to the cell that it is pointing for. They can receive these signals from other transmission states or from a confluent state.
```